README.md

```diff
@@ -13,6 +13,8 @@ To make changes to a distribution once deployed, update the origin path.
 For example, a version 1.0 project could be placed in a `/v1.0` directory, and the next version of the project placed in `/v1.1`.
 Updating the `origin_path` from `/v1.0` to `/v1.1` would invalidate the cache and redeploy the new distribution.
 
+To avoid using an ACM certificate and use the default cloudfront certificate, set `cloudfront_default_certificate` to `true` and do not provide `alias` or `acm_certificate_arn`.
```

Comment:
Should we move this to its own example? So the first example would be `Cloudfront distribution with custom ACM certificate` and your new example would be `Cloudfront distribution with default certificate`?